components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/EndpointRegistryConstants.java

```diff
@@ -59,4 +57,12 @@
 
     public static final String COLUMN_METADATA = "METADATA";
 
+    public static final String COLUMN_CREATED_BY = "CREATED_BY";
+
+    public static final String COLUMN_UPDATED_BY = "UPDATED_BY";
```

Comment:
in AM_DB tables we have been using UPDATED_TIME , CREATED_TIME. shall we use the same?

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/8550/commits/d29130c659d23aef2a47475d566e30fa30bcdc56